src/main/java/frc/robot/commands/SetIntakePosition.kt

```diff
@@ -4,26 +4,24 @@ import edu.wpi.first.math.geometry.Rotation2d
 import edu.wpi.first.math.trajectory.TrapezoidProfile
 import edu.wpi.first.wpilibj.Timer
 import edu.wpi.first.wpilibj2.command.Command
-import frc.robot.subsystems.intake.BallIntake
+import frc.robot.subsystems.intake.Intake
 
-class SetIntakePosition(private val setPosition: BallIntake.Position) : Command {
+class SetIntakePosition(private val setpoint: Rotation2d, private val intake: Intake) : Command {
```

Comment:
I didn't mention this in the issue but we should maybe replace this with a [trapezoid profile command](https://docs.wpilib.org/en/stable/docs/software/commandbased/profile-subsystems-commands.html#trapezoidprofilecommand).